backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/logging/impl/Log.java

```diff
@@ -107,4 +107,16 @@ SearchException conflictingFieldFormattersForSearch(String absoluteFieldPath,
 
 	@Message(id = 513, value = "A normalizer was set on field '%1$s' of type '%2$s', but fields of this type cannot be analyzed." )
 	SearchException cannotUseNormalizerOnFieldType(String fieldName, DataType fieldType);
+
+	@Message(id = 514, value = "Index '%2$s' requires multi-tenancy but backend '%1$s' does not support it.")
+	SearchException multiTenancyRequiredButNotSupportedByBackend(String backendName, String indexName);
+
+	@Message(id = 515, value = "Unknown multi-tenancy strategy '%1$s'.")
+	SearchException unknownMultiTenancyStrategyConfiguration(String multiTenancyStrategy);
+
+	@Message(id = 516, value = "Tenant identifier '%2$s' is provided whereas multi-tenancy is disabled for the backend '%1$s'.")
```

Comment:
Maybe it's just me, but I find the syntax "<what I got>, but <what I expected>" more effective at conveying the meaning "those two things are incompatible".
Here I could understand the message as "some unexpected situation arised even though you [the user] did everything right". Which would be definitely confusing, as I wouldn't know where to begin to fix the problem.
But once again, maybe it's just me...

Reply:
I think it's just you but changed anyway :).